doc/changelog/08-vernac-commands-and-options/16004-pr-Alizter-16004.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  Hint locality deprecations now raise an error
```

Comment:
I think that such a breaking change deserves a longer changelog. You should put some blinking warning somewhere stating that this is very backwards incompatible.